src/Compute/Compute/ChangeLog.md

```diff
@@ -19,9 +19,11 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
```

Comment:
Missing changelog for the following items:
https://msazure.visualstudio.com/One/_workitems/edit/7745035 
https://msazure.visualstudio.com/One/_workitems/edit/7745028
https://msazure.visualstudio.com/One/_workitems/edit/7745041
https://msazure.visualstudio.com/One/_workitems/edit/7745023

Reply:
Thank you for catching that. 7745028's changelog was already here